vuepress/docs/next/docs/README.md

```diff
@@ -10,7 +10,7 @@ only be available by building from source.
 
 # Entando Platform
 
-Entando is a **micro frontend platform for Kubernetes** that helps enterprises innovate faster with parallel development teams that have end-to-end autonomy across the entire stack.
+**Entando is the leading open source Application Composition Platform for Kubernetes**. It enables parallel development teams to innovate and accelerate new business technologies on a modern, cloud-native, and Kubernetes-native stack.
```

Comment:
1. We should keep "capabilities" instead of "technologies". Technologies implies things like k8s, react, http, etc. The semantics for "capabilities" is more about the features of a system that a business needs, hence Packaged Business Capabilities which is one of the types of entries in the Hub.

2. Maybe "enables parallel teams to accelerate development and innovation of business capabilities on..."  

3. Let's just bold "Application Composition Platform" in the first sentence to bring attention to that.

Reply:
“Entando is the leading open source Application Composition Platform for Kubernetes. It enables parallel development teams to innovate and accelerate the expansion of business capabilities on a modern, cloud-native, and Kubernetes-native stack.”

@jyunmitch and i were going back and forth and this is my vote!

Reply:
got it!